packages/extension-chakra-storefront/src/components/header/index.jsx

```diff
@@ -257,7 +257,7 @@ const Header = ({
                                         <Popover.Body css={styles.dropdownMenuBody}>
                                             <Popover.Header pb={1}>
                                                 <Popover.Title>
-                                                    <Heading as="h2" fontWeight={700} size="lg">
+                                                    <Heading as="h2" size="lg">
```

Comment:
Heading is now default to fontWeight: bold, no need to customise from the component